app/scripts/models/account.js

```diff
@@ -96,7 +96,8 @@ define([
             self.set('verified', verified);
           }, function () {
             // Ignore errors; we'll just fetch again when needed
-          });
+            // TODO Remove once https://github.com/eslint/eslint/issues/3279 gets fixed
```

Comment:
> The issue is getting fixed soon

The linked issue in that 3279 issue was opened since February, so I'm not sure how soon that will be fixed